<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCanCfgForm</class>
 <widget class="QWidget" name="CCanCfgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>CAN</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>350</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="can1ActiveChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aktywny CAN1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="can2ActiveChk">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aktywny CAN2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="canAcceptCmd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Akceptowanie poleceń sterujących</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>115</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nr urządzenia:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="canNoSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prędkość:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="canBaudCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>145</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>25 kbps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>125 kbps</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="canRepeaterActiveChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CAN Repeater</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="canMonEnableChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CAN MON</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
